docs/reference/cainjector.rst

```diff
@@ -10,3 +10,6 @@ APIService resources annotated with:
   Injects the cluster CA.
 * ``cert-manager.io/inject-ca-from: <NAMESPACE>/<CERTIFICATE>``  
   Injects the CA from the specified :doc:`certificate </reference/certificates>`.
+* ``cert-manager.io/inject-ca-from-secret: <NAMESPACE>/<CERTIFICATE>``  
```

Comment:
maybe change `CERTIFICATE` to `SECRET` here? 😄

Reply:
Year, thank you 😅 
I’ll fix now.